openaerostruct/structures/spatial_beam_setup.py

```diff
@@ -46,3 +46,13 @@ def setup(self):
                 promotes_inputs=["nodes", "A_int"],
                 promotes_outputs=["fuel_vols"],
             )
+        # ================================================
+        # Adding Tsai Wu wingbox here
+        # ================================================
+        elif surface["fem_model_type"] == "tsaiwu_wingbox":
+            self.add_subsystem(
+                "fuel_vol",
+                WingboxFuelVol(surface=surface),
+                promotes_inputs=["nodes", "A_int"],
+                promotes_outputs=["fuel_vols"],
+            )
```

Comment:
What is this?

Reply:
`tsaiwu_wingbox` is no longer a valid value for `fem_model_type`, we use `wingbox` for both the composite and metallic wingbox, so these lines can simply be removed